composer.json

```diff
@@ -1,46 +1,53 @@
 {
-	"name": "phpab/phpab",
-	"type": "library",
-	"description": "A PHP library to create A/B tests.",
-	"keywords": [
-		"Phpab",
-		"Ab test",
-		"abtest"
-	],
-	"homepage": "https://github.com/phpab/phpab",
-	"license": "MIT",
+    "name": "phpab/phpab",
+    "type": "library",
+    "description": "A PHP library to create A/B tests.",
+    "keywords": [
+        "Phpab",
+        "Ab test",
+        "abtest"
+    ],
+    "homepage": "https://github.com/phpab/phpab",
+    "license": "MIT",
     "authors": [
-		{
```

Comment:
"license" and "authors" weren't at the same level and it wasn't consistant (spaces and tabs)
I did an automatic indentation. It seems correct to me now, isn't it?